packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.spec.js

```diff
@@ -57,8 +57,13 @@ describe('Component > SingleImageViewerContainer', function () {
       const subject = {
         id: 'test',
         locations: [
+          { 'image/jpeg': 'https://some.domain/image.jpg' },
+          { 'image/jpeg': 'https://some.domain/image.jpg' },
           { 'image/jpeg': 'https://some.domain/image.jpg' }
-        ]
+        ],
+        metadata: {
+          default_frame: "0"
```

Comment:
Doesn't hurt, but why change the single image viewer testing subject from a single image subject to a multiple image subject? Not sure if there could be unintended consequences down the road, like if a subject naturally doesn't have the noted metadata but somehow our tests expect it.

Reply:
I have the same question. The single image viewer can only receive a subject with a single image location as a prop, so why change the tests?

Reply:
Likely this is a mistake.